index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="menu-wrap">
+      <div class="logo">
+        <span class="brand">Saw Mon Han</span>
+        <input type="checkbox" class="toggler" />
+        <div class="hamburger"><div></div></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <main>
```

Comment:
- [ ] I think it would be better for this section for the container to use a ```<section>``` tag to replace the ```<main>``` tag. 😹 
* It will represent a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. 👍 👍 

- [ ]  Kindly try to add a visible background image as provided by the Figma design, to follow the design as much as possible 🙏 

![image](https://user-images.githubusercontent.com/102757126/212693651-026f5458-ec34-4559-8df6-1e9473505371.png)